package.json

```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@nomiclabs/hardhat-truffle5": "^2.0.0",
+    "@uniswap/v3-core": "^1.0.1",
```

Comment:
As said before, please copy the contract into our own folder and remove the dependency.